src/db/impl/pouch.ts

```diff
@@ -0,0 +1,90 @@
+import {DBOptions} from "../interface";
+
+import PouchDB from "pouchdb-core";
+import MemoryAdapter from "pouchdb-adapter-memory";
+import AbstractDB, {SearchOptions} from "./abstract";
+
+PouchDB.plugin(MemoryAdapter);
+
+const BASE_REVISION = "1";
+
+export class PouchDb extends AbstractDB {
+
+  private db: PouchDB.Database;
+
+  public constructor(opts: DBOptions) {
+    super();
+    this.db = new PouchDB(
+      opts.name || 'lodestar-beaconchain',
+      {
+        adapter: 'memory',
+      }
+    );
+  }
+
+  public start(): Promise<void> {
+    return null;
+  }
+
+  public stop(): Promise<void> {
+    return this.db.close();
+  }
+
+  public clean(): Promise<void> {
+    return this.db.destroy();
+  }
+
+  public async batchDelete(items: any[]): Promise<any> {
+    const deletitions = [];
```

Comment:
could you fix this mis-spelling?

Reply:
Will address those two issues, sometimes tomorrow